src/Components/Nav/Nav.scss

```diff
@@ -13,16 +13,31 @@ nav {
   opacity: .85;
 }
 
+.favorites-link {
+  width: 100%;
+  text-decoration: none;
+}
+
+.nav-favorites-img {
+  height: 30px;
+  width: 30px;
+  margin: 0 10px;
+}
+
 .btn-favorites {
   opacity: .8;
-  width: 13%;
+  width: 100%;
   padding: 5px;
   border-radius: 10px;
   background-color: black;
   color: white;
   font-size: 1.5em;
+  margin-bottom: 5px;
   outline: 0;
   cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
If we have time we should try and re format our SCSS to incorporate nesting, might make it a little easier to read?